src/Components/ProfilePage.js

```diff
@@ -1,13 +1,42 @@
 import React, { Component } from 'react';
+import { MDBContainer, MDBRow, MDBCol, MDBBtn, MDBIcon, MDBCardBody } from "mdbreact";
 import './styles/ProfilePage.css'
 
 class ProfilePage extends Component {
     constructor(){
         super();
+        this.name = "Lulu Liu";
+        this.biography = "Et harum quidem rerum facilis est et expedita distinctio. Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus. Temporibus autem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusandae. Itaque earum rerum hic tenetur a sapiente delectus, ut aut reiciendis voluptatibus maiores alias consequatur aut perferendis doloribus asperiores repellat. ";
+        this.program = "SYDE - 3A";
+        this.socialMedia = "@LuluLiu";
+        this.avatar = "https://workmacro.com/wp-content/uploads/2018/02/1-by-1-1024x1024.png";
```

Comment:
@zhengwin & I will have to define a common pattern to store avatar files locally in react-app or fetched from server. That way we control management of assets. 

Add a quick TODO comment at the top saying to replace these fields with dynamic values :) .

Reply:
In react, we don't store class variables as "this.variable =". We store all dynamic variables within a built-in variable called "this.state" and we modify state with the function "this.setState()". Look at landingpage.js and onBoarding.js for examples. 

your new constructor function should look like this:
constructor() {
    super();
    this.state = {
        biography: "Et harum quidem rerum facilis est et expedita distinctio...",
        program: "SYDE - 3A",
        socialMedia: "@LuluLiu",
       avatar: "https://workmacro.com/wp-content/uploads/2018/02/1-by-1-1024x1024.png"
    }
}

Reply:
The profile page looks great, good job :D